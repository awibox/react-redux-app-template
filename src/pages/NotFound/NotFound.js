import React from 'react';
// Components
import { Card } from 'tigerspack';
// Styles
import styles from './NotFound.scss';

const NotFound = () => (
  <Card theme={'light'} className={styles.notFound}>
    <div className={styles.code}>404</div>
    <p>We couldn&apos;t find this page.</p>
    <p>The page you are looking for does not exists.</p>
  </Card>
);
export default NotFound;
