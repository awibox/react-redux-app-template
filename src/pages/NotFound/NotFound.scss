@import '../../styles/variables';

.notFound {
  display: flex;
  flex-flow: column;
  font-size: 20px;
  text-align: center;
  p {
    padding-bottom: $defaultIndent;
  }
}
.code {
  font-size: 42px;
  line-height: 50px;
  color: #333;
  padding: $defaultIndent*2;
}
