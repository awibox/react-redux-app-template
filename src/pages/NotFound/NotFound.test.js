import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound', () => {
  it('should render correctly', () => {
    const component = shallow(<NotFound />);
    expect(component).toMatchSnapshot();
  });
});
