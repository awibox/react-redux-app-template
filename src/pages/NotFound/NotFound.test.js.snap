// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
<Card
  className="notFound"
  padding={15}
  theme="light"
>
  <div
    className="code"
  >
    404
  </div>
  <p>
    We couldn't find this page.
  </p>
  <p>
    The page you are looking for does not exists.
  </p>
</Card>
`;
